changelog/upgrade-notes.md

```diff
@@ -1,5 +1,7 @@
 ## Upgrade notes
 
+### v3.14.0
+
```

Comment:
Ok.  The important part is that the upgrade notes have a header for the current release - so future changes get added above instead of mixed into the previous notes.  I changed the [release procedure docs](https://github.com/openlayers/ol3/wiki/Release-Procedure/_compare/01d8f2998bdc39af9e1e1dbc43855ca1985a602e) to try to clarify this.

If someone adds a header between releases and it looks like `### Next Release`, then you need to edit it to read `### v3.1.0` (assuming you are releasing `v3.1.0`).

If it already reads `### v3.1.0`, you don't need to do anything.  However, guessing the next release version number is a mistake since a patch release may come before the next minor release.  So people should *not* try to add a header between releases that includes the next release number.

Hope this makes sense.